components/wishlist-page/wishlist-page.html

```diff
@@ -0,0 +1,165 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="wishlist-page.css">
+    <script src="https://kit.fontawesome.com/9c98d67da5.js" crossorigin="anonymous"></script>
+    <title>Quartz</title>
+</head>
+
+<body>
+    <header class="header-nav">
+        <div class="navbar">
+
+            <div class="logo">
+                <h1><i class="fab fa-artstation"></i>Quartz</h1>
+            </div>
+
+            <nav class="nav">
+                <ul class="nav-links">
+                    <li><a href="/index.html">HOME</a></li>
+                    <li><a href="/components/product-page/product-page.html">PRODUCTS</a></li>
+                    <li><a href="/">ACCOUNT</a></li>
+                    <li><a href="/components/wishlist-page/wishlist-page.html"> <i class="fas fa-heart"></i></a></li>
+                    <li><a href="/components/cart-page/cart-page.html"><i class="fas fa-shopping-bag"></i></a></li>
+                </ul>
+            </nav>
+
+        </div>
+    </header>
+
+    <h1 class="heading-my-wishlist">My Wishlist &#x2764;</h1>
+
+    <section class="wishlist-card-container">
+
+        <div class="wishlist-card">
+            <div class="card card5 card7">
+                <section class="img-container">
+                    <img class="card-image" alt="card-image"
+                        src="https://images.unsplash.com/photo-1583743814966-8936f5b7be1a?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxzZWFyY2h8Nnx8dCUyMHNoaXJ0fGVufDB8fDB8fA%3D%3D&auto=format&fit=crop&w=400&q=60">
```

Comment:
try to store photos inside  a folder(assets) in your files and then use it

Reply:
make a folder or image then use those image and rename those images